src/Gui/Quarter/Quarter.cpp

```diff
@@ -121,22 +121,23 @@
   \subpage examples
 */
 
-/*!
-  \page examples More Examples
-
-  The examples code is included in Quarter and can be found in the
-  src/examples subdirectory.
-
-  \subpage directui
-
-  \subpage dynamicui
-
-  \subpage inheritui
-
-  \subpage mdi
-
-  \subpage examiner
-*/
+// These pages are not exitant in this copy of Quarter
```

Comment:
`These pages are not existant in this copy of Quarter`
or
`These pages do not exist in this copy of Quarter`
Think the 2nd one makes more sense. 
Question though: why does it say immediately below that 
```
//  The examples code is included in Quarter and can be found in the
```